system/CLI/BaseCommand.php

```diff
@@ -150,7 +150,7 @@ protected function call(string $command, array $params = [])
 	{
 		// The CommandRunner will grab the first element
 		// for the command name.
-		array_unshift($params, $command);
+		array_push($params, $command);
```

Comment:
This seems like a pretty major change with potential issues. Can you elaborate why?

Reply:
When I tried to call another commands on `CreateAll.php` like `$this->call('make:entity', [$name]);` then `array_unshift` put the command itself first, creating at the end an unexpected `make:entity.php` file.

I searched where it was previously used, and it was in `CreateMigration.php` in which an associative array was the parameter (No matter the order). So I just pushed the command at the end in case you would need it in a future.